### PR TITLE
Fixed 'bitmask' attribute definition 

### PR DESCRIPTION
**Pull request by BazookaJoe1900**
so it will pass xml validation without error

### CODE REVIEW

File: generator/mavschema.xsd [hunk 1]
```diff
@@ -230,6 +230,9 @@
     </xs:complexType>
 </xs:element>
 
+<!-- definition entry elements attributes (like the ones used on MAV_CMD for example) -->
```
Comment (peterbarker) [inline generator/mavschema.xsd:233]: Should this go up near the top where all of the attributes are defined?

Comment (BazookaJoe1900) [reply generator/mavschema.xsd:233]: I kind of copy-past how it done before, I didn't want to change too much

**Comment by BazookaJoe1900** [comment]
Another commit added, to print errors in schema for future erros

**Comment by peterbarker** [comment]
Merged, thanks

**Comment by hamishwillee** [comment]
Something still going on, but too tired to properly think about it tonight. The tests are failing with 
```
MAV_CMD
MAV_CMD_RESET_MPPT
MAV_CMD_PAYLOAD_CONTROL
Note: message ASLCTRL_DATA is longer than 64 bytes long (106 bytes), which can cause fragmentation since many radio modems use 64 bytes as maximum air transfer unit.
Note: message FW_SOARING_DATA is longer than 64 bytes long (110 bytes), which can cause fragmentation since many radio modems use 64 bytes as maximum air transfer unit.
Validating message_definitions/v1.0/common.xml
ERROR: /home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2048:0:ERROR:SCHEMASV:SCHEMAV_CVC_COMPLEX_TYPE_3_2_1: Element 'param', attribute 'name': The attribute 'name' is not allowed.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2049:0:ERROR:SCHEMASV:SCHEMAV_CVC_COMPLEX_TYPE_3_2_1: Element 'param', attribute 'name': The attribute 'name' is not allowed.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2050:0:ERROR:SCHEMASV:SCHEMAV_CVC_COMPLEX_TYPE_3_2_1: Element 'param', attribute 'name': The attribute 'name' is not allowed.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2122:0:ERROR:SCHEMASV:SCHEMAV_ELEMENT_CONTENT: Element 'wip': This element is not expected. Expected is one of ( description, param ).
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2137:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2138:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2139:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2147:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2148:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2149:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2157:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2158:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2159:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2210:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2211:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2212:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2213:0:ERROR:SCHEMASV:SCHEMAV_CVC_DATATYPE_VALID_1_2_1: Element 'param', attribute 'reserved': 'True' is not a valid value of the atomic type 'xs:boolean'.
/home/travis/build/mavlink/mavlink/message_definitions/v1.0/common.xml:2217:0:ERROR:SCHEMASV:SCHEMAV_ELEMENT_CONTENT: Element 'wip': This element is not expected. Expected is one of ( description, param ).
```

**Comment by BazookaJoe1900** [comment]
@hamishwillee Where have you seen this errors?

**Pull request merged**